awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.test.jsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+
+import {
+  mountWithContexts,
+  waitForElement,
+} from '../../../../testUtils/enzymeHelpers';
+
+import { InstanceGroupsAPI } from '../../../api';
+import InstanceGroupList from './InstanceGroupList';
+
+jest.mock('../../../api/models/InstanceGroups');
+
+const instanceGroups = {
+  data: {
+    results: [
+      {
+        id: 1,
+        name: 'Foo',
+        type: 'instance_group',
+        consumed_capacity: 10,
+        summary_fields: { user_capabilities: { edit: true, delete: true } },
```

Comment:
To prevent a prop type error, we need to add a `url` key and value to each of these result items.

Reply:
Updated.